ConwaysGameOfLife.Tests/CellShould.cs

```diff
@@ -54,5 +54,15 @@ public void WhenCellStateAndLocationAreSame_BeEqual()
             Assert.Equal(expected, result);
         }
 
+        [Fact]
+        public void WhenUniverseRegenerated_SwitchState()
+        {
+            var cell = new Cell(CellState.Dead, 0, 0);
+            var expected = new Cell(CellState.Alive, 0, 0);
+
+            var result = cell.SwitchCellState(cell);
+
+            Assert.Equal(expected, result);
+        }

```

Comment:
Do we want the Cell to know how to regenerate itself? Or is it "dumb" and some higher class is responsible for regenerating the Universe as a whole, and thereby the Cells?

Reply:
@Kedoodle this was my doing . I interpreted this as toggling the state of the cell so asked Linda to move the method here.
@lindajohnstone  your options:
1. If the cell can toggle its state, keep the method here
2. If the universe is responsible for regenerating itself as a whole and thereby the cells, rename the test to better reflect that a new cell is created with a different state and move this method back to the universe. To paraphrase uncle Bob, when naming a test, it does what it says and says what it does.

Reply:
no worries. the name implies the universe being regenerated, but there is no universe instance or regeneration of that instance.